### PR TITLE
feat(combo): add combo binding example

### PR DESCRIPTION
**Pull request by ViktorSlavov**
https://github.com/IgniteUI/igniteui-docfx/issues/1480

### CODE REVIEW

**Comment by dafo** [comment]
When selecting more than one item the selection is cropped:

Also, the items are not grouped:

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <div name="codes">{{ selectedValueKey }}</div>
+    </div>
+</div>
+<div class="combo-section">
+    <button igxButton="raised" (click)="selectTopItems(false)">Select Most Populated</button>
+    <igx-combo #noValueKey [data]="towns" displayKey="name" groupKey="country" [(ngModel)]="selectedNoValueKey">
+    </igx-combo>
+    <label for="cities">Selected Cities:</label>
+    <div name="cities" class="cities-container">
+        <div class="card-wrapper" *ngFor="let city of selectedNoValueKey">
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]:
```suggestion
```
No need for this. You **_can_** style the card just fine:
```sass
.cities-container {
  // ...
  igx-card {
    margin: 8px 8px 8px 0;
  }
}
```

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            <igx-card type="outlined" width="120px">
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]: No need for the width, not sure it even does something. Also per above, move the ngFor here:
```suggestion
            <igx-card type="outlined" *ngFor="let city of selectedNoValueKey">
```

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                <igx-card-header>
+                    <h5 igxCardHeaderTitle>{{ city.name }}</h5>
+                    <h5 igxCardHeaderSubtitle>{{ city.country }}</h5>
+                    <div class="card-id">{{ city.id }}</div>
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]: This is a bit too attention-grabby for me. Just the text seems to be placed nicely:
```suggestion
                    {{ city.id }}
```

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
+:host {
+    display: flex;
+    width: 100%;
+    flex-flow: row;
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]:
```suggestion
    flex-flow: row wrap;
```

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<div class="combo-section">
+    <button igxButton="raised" (click)="selectTopItems(true)">Select Most Populated</button>
+    <igx-combo #withValueKey [data]="towns" valueKey="id" displayKey="name" groupKey="country"
+        [(ngModel)]="selectedValueKey">
+    </igx-combo>
+    <div>
+        <label for="codes">Selected IDs:</label>
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]: Same goes here, actually.. maybe even an actual title instead so we can skip the font style too?

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        <div name="codes">{{ selectedValueKey }}</div>
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]:
```suggestion
        <div>{{ selectedValueKey }}</div>
```

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+.combo-section {
+    margin: 24px;
+    width: 320px;
+
+    label {
+        font-weight: 800;
+    }
+}
+
+.model-container {
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]: Leftover style?

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    max-width: 100%;
+    height: 56px;
+    overflow: hidden;
+    margin-bottom: 8px;
+    padding-bottom: 4px;
+    border-bottom: 1px solid black;
+}
+.cities-container {
+    display: flex;
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]:
```suggestion
```
No need for explicit flex + column on this container

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    flex-flow: column;
+    max-height: 200px;
+    border-bottom: 1px solid #a3a3a3;
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]: drop that border, pretty please

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+button {
+    margin: 8px 0px;
+}
+
+.combo-section {
+    margin: 24px;
+    width: 320px;
+
+    label {
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]: not needed anymore

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+        font-weight: 800;
+    }
+}
+
+.cities-container {
+    max-height: 200px;
+    overflow-y: auto;
+    margin: 8px 0px;
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]:
```suggestion
```

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+    igx-card {
+        padding: 8px;
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]: Padding was on the wrapper, for the card is margin (see unresolved comment from last review):
```suggestion
        margin: 8px 8px 8px 0;
```

File: src/app/lists/combo/combo-binding/combo-binding.component.scss [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+        overflow: visible;
```
Comment (damyanpetev) [inline src/app/lists/combo/combo-binding/combo-binding.component.scss]:
```suggestion
```

File: src/app/lists/combo/combo-binding/combo-binding.component.ts [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+    templateUrl: "combo-binding.component.html",
+    styleUrls: ["combo-binding.component.scss"]
+})
+export class ComboBindingComponent {
+    @ViewChild("withValueKey", { read: IgxComboComponent, static: false })
+    public comboValueKey: IgxComboComponent;
+    @ViewChild("noValueKey", { read: IgxComboComponent, static: false })
+    public comboNoValueKey: IgxComboComponent;
+
+    public towns: ICity[] = CITIES;
```
Comment (wnvko) [inline src/app/lists/combo/combo-binding/combo-binding.component.ts]:
```suggestion
    public cities: ICity[] = CITIES;
```

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 15]
```diff
@@ -0,0 +1,23 @@
+<div class="combo-section">
+    <igx-combo #withValueKey [data]="towns" valueKey="id" displayKey="name" groupKey="country"
```
Comment (wnvko) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]:
```suggestion
    <igx-combo #withValueKey [data]="cities" valueKey="id" displayKey="name" groupKey="country"
```

File: src/app/lists/combo/combo-binding/combo-binding.component.html [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+        [(ngModel)]="selectedValueKey">
+    </igx-combo>
+    <button igxButton  (click)="selectFavorites(true)">Select Favorites</button>
+    <h6>Selected Cities:</h6>
+    <div>{{ selectedValueKey }}</div>
+</div>
+<div class="combo-section">
+    <igx-combo #noValueKey [data]="towns" displayKey="name" groupKey="country" [(ngModel)]="selectedNoValueKey">
```
Comment (wnvko) [inline src/app/lists/combo/combo-binding/combo-binding.component.html]:
```suggestion
    <igx-combo #noValueKey [data]="cities" displayKey="name" groupKey="country" [(ngModel)]="selectedNoValueKey">
```

**Pull request merged**